src/Tabs/Tab.js

```diff
@@ -47,22 +48,31 @@ export const styleSheet = createStyleSheet('MuiTab', (theme) => {
     fullWidth: {
       flex: '1 0 0',
     },
+    labelContainer: {
+      paddingTop: 6,
+      paddingBottom: 6,
+      paddingLeft: 12,
+      paddingRight: 12,
+      [theme.breakpoints.up('md')]: {
+        paddingLeft: 24,
```

Comment:
What about using the `theme.spacing.unit * X`? I'm not sold on the issue. I want to get your point of view.